Assets/UI Toolkit/Scripts/Components/Icon.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Netherlands3D.UI;
+using Netherlands3D.UI.ExtensionMethods;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Netherlands3D.UI.Components
+{
+    [UxmlElement]
+    public partial class Icon : VisualElement
+    {
+        public enum IconImage
+        {
+            Plus,
+            Map,
+            Folder,
+            Trash,
+        }
+
+        public enum IconColor
```

Comment:
A supported list of images, these map onto the `.icon.tint-X` class definitions in the Icon.uss; using an enum will give a list of options in the UI Builder to pick from, and for the devs to know which to choose if they use code